src/components/app_catalog/detail/cluster_picker.js

```diff
@@ -18,28 +18,28 @@ const ClusterList = styled.div`
 
 const Cluster = styled.div`
   align-items: center;
-  border-radius: ${theme.BORDER_RADIUS};
+  border-radius: ${theme.border_radius};
```

Comment:
The theme doesn't actually need to be imported, it is in the context of all styled components (so all of them receive it as a prop).

Usage is like:
```css
border-radius: ${({ theme }) => theme.border_radius};
```

Importing it separately will make the components not respond to dynamic theme updates

Reply:
Interesting, thanks!

Reply:
Changed.